ts/features/idpay/initiative/details/components/BonusCardComponent.tsx

```diff
@@ -0,0 +1,176 @@
+import { Text, View } from "native-base";
+import * as React from "react";
+import { Image, Platform, StyleSheet } from "react-native";
+import {
+  InitiativeDTO,
+  StatusEnum
+} from "../../../../../../definitions/idpay/wallet/InitiativeDTO";
+import { makeFontStyleObject } from "../../../../../components/core/fonts";
+import { H2 } from "../../../../../components/core/typography/H2";
+import { H5 } from "../../../../../components/core/typography/H5";
+import { IOColors } from "../../../../../components/core/variables/IOColors";
+import TypedI18n from "../../../../../i18n";
+import { formatDateAsLocal } from "../../../../../utils/dates";
+import bonusVacanzeWhiteLogo from "../../../../../../img/bonus/bonusVacanze/logo_BonusVacanze_White.png";
+import BonusCardSVG from "../../../../../../img/idpay/bonus_bg_svg.svg";
+
+type Props = InitiativeDTO;
+
+const opaqueBorderColor = IOColors.bluegreyDark;
+
+const styles = StyleSheet.create({
+  flexFull: {
+    flex: 1
+  },
+  imageFull: {
+    resizeMode: "stretch",
+    zIndex: -1
+  },
+  row: {
+    flexDirection: "row",
+    alignItems: "center",
+    textAlign: "center"
+  },
+  spaced: {
+    justifyContent: "space-between"
+  },
+  bottomBonusIcon: {
+    flex: 1,
+    justifyContent: "flex-end"
+  },
+  paddedMainContent: {
+    zIndex: 1,
+    padding: 16,
+    width: "100%",
+    height: "100%",
+    position: "absolute"
+  },
+  Amount: {
+    lineHeight: 32,
+    fontSize: 24,
+    color: IOColors.white,
+    ...makeFontStyleObject("Bold", undefined, "TitilliumWeb")
+  },
+  logo: {
+    resizeMode: "contain",
+    height: 65,
+    width: 65
+  },
+  consumedOpacity: {
+    opacity: 0.5
+  },
+  shadowBox: {
+    marginBottom: -13,
+    borderRadius: 8,
+    borderTopWidth: 10,
+    borderTopColor: opaqueBorderColor,
+    height: 15,
+    width: "100%",
+    maxWidth: 343
+  }
+});
+
+const BonusCardComponent = (props: Props) => {
```

Comment:
```suggestion
const InitiativeCardComponent = (props: Props) => {
```

please also use the term Initaitive in the rest of the code.